forward_engineering/helpers/referencesHelper.js

```diff
@@ -1,12 +1,13 @@
 /**
  * Get the definition name from the reference path
  *
- * @param {Object} param0
- * @param {string} param0.referencePath - The reference path, separated by '/', where the definition name is the last element.
+ * @param {object} param0
+ * @param {string} param0.referencePath - The reference path, separated by '/', where the definition name is the last
+ *   element.
  * @returns {string} - The definition name.
  */
 function getDefinitionNameFromReferencePath({ referencePath = '' }) {
```

Comment:
We got two identical files `referenceHelper.js` and `referencesHelper.js`.
It's worth removing one of them in this PR.